CHANGES

```diff
@@ -1,3 +1,9 @@
+0.25.2
+------
+
+* Fixed error messages when matches fail. See #704
+* Delete `_create_key_val_str` which was introduced for the compatibility between python2 (not supported anymore) and python3.
```

Comment:
I would not talk about internal method. Instead, please briefly describe what changed for the end user